package.json

```diff
@@ -67,12 +67,8 @@
     "update-doc-src-from-readme": "cross-env-shell node ./scripts/from-readme/index.js",
     "update-readme": "cross-env-shell node ./scripts/update-readme-files.js",
     "add-desc-to-src": "cross-env-shell node ./scripts/add-description-to-src.js",
-    "watch": "cross-env-shell lerna exec --parallel -- rollup -w -c=`pwd`/rollup.config.js"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run update-readme && lint-staged && npm run check-deps"
-    }
+    "watch": "cross-env-shell lerna exec --parallel -- rollup -w -c=`pwd`/rollup.config.js",
+    "prepare": "husky install"
```

Comment:
Why did we choose `prepare`? Seems it's only relevant on `postinstall`, not on publish etc

Reply:
It was done by husky migration tool. I think here you could find the explanation behind this decision:
https://github.com/typicode/husky/issues/884

Reply:
Great, thanks :)

> Unlike postinstall, the prepare command does not run when a NPM package is installed as a dependency in another project.

That's reason enough :+1: